SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -421,6 +421,8 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 		Connection->GetCoordinator().CreateSubView(SpatialConstants::ACTOR_NON_AUTH_TAG_COMPONENT_ID, ActorFilter, RefreshCallbacks);
 	const SpatialGDK::FSubView& TombstoneActorSubview = Connection->GetCoordinator().CreateSubView(
 		SpatialConstants::TOMBSTONE_TAG_COMPONENT_ID, SpatialGDK::FSubView::NoFilter, SpatialGDK::FSubView::NoDispatcherCallbacks);
+	DebugActorSubView =
+		&Connection->GetCoordinator().CreateSubView(SpatialConstants::GDK_DEBUG_COMPONENT_ID, ActorFilter, RefreshCallbacks);
```

Comment:
Is it definitely the case that the debug system doesn't want actors with tombstone components? It makes sense, but I'm unconvinced these were previously filtered out before.